spk/fuse-exfat/Makefile

```diff
@@ -0,0 +1,24 @@
+SPK_NAME = fuse-exfat
+SPK_VERS = 1.2.1
+SPK_REV = 2  
+SPK_ICON = src/fuse.png
+
+DEPENDS = cross/$(SPK_NAME) cross/exfat-utils
+
+MAINTAINER = Roman Tymoshyk 
```

Comment:
Probably you'll want to put SynoCommunity or a username here (based on the assumption that the repo can handle usernames)